models.py

```diff
@@ -203,7 +203,7 @@ def complete(self, note='', user=None):
         )
 
         self.typesetter_note = note
-
+        self.accepted = timezone.now()
```

Comment:
We should respect the original accepted date where there is one instead of overriding it.